snowflake/assets/dashboards/organization_metrics.json

```
@@ -0,0 +1,982 @@
+                            "width": 6,
+                            "x": 6,
+                            "y": 2
+                        }
+                    }
+                ]
+            },
+            "id": 3702666235793118,
+            "layout": {
+                "height": 5,
+                "width": 12,
+                "x": 0,
+                "y": 21
+            }
+        },
+        {
+            "definition": {
+                "background_color": "vivid_blue",
+                "layout_type": "ordered",
+                "show_title": true,
+                "title": "Usage in Currency",
+                "type": "group",
+                "widgets": [
+                    {
+                        "definition": {
+                            "requests": [
+                                {
+                                    "formulas": [
+                                        {
+                                            "formula": "query1"
+                                        }
+                                    ],
+                                    "queries": [
+                                        {
+                                            "aggregator": "avg",
+                                            "data_source": "metrics",
+                                            "name": "query1",
+                                            "query": "avg:snowflake.organization.currency.usage{$billing_account,$scope} by {usage_type}"
+                                        }
+                                    ],
+                                    "response_format": "scalar"
+                                }
+                            ],
+                            "title": "Credits Charged per Usage Type",
+                            "type": "treemap"
+                        },
+                        "id": 2752166910614806,
+                        "layout": {
+                            "height": 2,
+                            "width": 6,
+                            "x": 0,
+                            "y": 0
+                        }
+                    },
+                    {
+                        "definition": {
+                            "requests": [
+                                {
+                                    "formulas": [
+                                        {
+                                            "formula": "query1"
+                                        }
+                                    ],
+                                    "queries": [
+                                        {
+                                            "aggregator": "avg",
+                                            "data_source": "metrics",
+                                            "name": "query1",
+                                            "query": "avg:snowflake.organization.currency.usage_in_currency{$billing_account,$scope} by {usage_type}"
```

Same here, let's add the `currency` tag to the query aggregation scope. 
```suggestion
                                            "query": "avg:snowflake.organization.currency.usage_in_currency{$billing_account,$scope} by {currency, usage_type}"
```